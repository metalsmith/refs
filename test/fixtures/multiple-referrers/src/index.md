---
refs:
  site: ./site.yml
  about: ./about.md
---