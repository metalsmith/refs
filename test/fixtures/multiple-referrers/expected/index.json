{
  "refs": {
    "site": {
      "id": "site-data",
      "title": "Metalsmith bare starter",
      "lang": "en-US",
      "generator": "Metalsmith",
      "generatorurl": "https://metalsmith.io",
      "nav": [
        {
          "id": "home",
          "label": "Home",
          "uri": ""
        },
        {
          "id": "about",
          "label": "About",
          "uri": "about"
        }
      ]
    },
    "about": {
      "id": "about.md"
    }
  },
  "id": "index.md"
}